### DISCUSSION TITLE
Is there any reason why Inure won't verify?

### DISCUSSION THREAD

**Discussion by g1zm02k** [Hamza417/Inure#348] [Q&A] [answered]
Hey Hamza,

I bought the unlocker for Inure yesterday and I can't for the life of me get it to actually register...

Positional worked first time, and Inure registered straight away on my Samsung Galaxy tablet, but it's having nothing to do with my Poco F4 for some reason (Xiaomi are turning out to be more trouble than they're worth).

Inure is basically just stuck with a 'Verifying License' banner along the bottom that won't go away until I swipe it down; and then it reverts to the 15 day trial. Posistional worked fine, even various removing and reinstalling in various orders did nothing to help it; all my other apps verified fine so it's just this one. Phone: Poco F4, on MIUI Global 14.0.4.

Here's some grabs if they're of any use:
[Unlocker is (alledgedly) Fine](https://i.imgur.com/pkrxvAv.jpg)
[Bottom Banner](https://i.imgur.com/cdFcPqP.jpg)
[Fifteen Days Notice](https://i.imgur.com/8IFupsr.jpg)
[Phone Info](https://i.imgur.com/OH5wjwS.jpg)

Let me know if there's anything else I can provide; and preferably how to go about getting the relevant info just in case.

Thanks in advance for any light you can shed on this.

Regards,

Will

**Comment by Hamza417** [comment]
It's possible that the Xiaomi's MiUI/HyperOS is explicitly blocking the unlocker from responding to the Inure's request. Can you try to disable any battery optimization for the Unlocker app and check if background connection is allowed and then try verifying it again and see if it works?

**Comment by g1zm02k** [comment]
Tried that. I've turned battery optimisations off for both the unlocker and the app itself - even turned off _Adguard_ to see if that was blocking the signal. Nothing is blocked, everything is allowed. This is literally the first app where I've had anything like this happen...

Hmm, I've just un/re-installed the unlocker again, and I got a very brief pop-up saying it keeps stopping.

**Comment by g1zm02k** [comment, marked as answer]
I've sorted it!

By pulling both up apps up together and spamming the unlocker's 'retry verification' option while flipping back and forth, it's finally relented and unlocked.

Weird thing to happen, but I'm good for now I guess.

Thanks for your quick response, and sorry for wasting your time - I appreciate it!
